charts/fint-oneroster-rest-provider/values-oneroster-rest-provider-agder.yaml

```diff
@@ -0,0 +1,148 @@
+replicaCount: 1
+nameOverride: oneroster-rest-provider-agder
+environment: beta
+onePasswordItem: 
+  name: oneroster-agder
+  create: false
+
+service:
+  type: LoadBalancer
+  port: 8080
+  
+deployment:
```

Comment:
is "deployment" required in this file? could you not have had the values ​​without?

Reply:
Used to split the values file into service and deployment sections for better readability